docs/README-secure-boot.md

```diff
@@ -113,6 +115,10 @@ A few variables can be used to configure this feature, including:
 
 The complete list of variables can be found in the `tdx-signed-fit-image.inc` file.
 
+### Configuring FIT image signing / known issues
+
+- On the **Verdin AM62** SoM, some of the configuration variables (e.g. `UBOOT_SIGN_KEYDIR`, `UBOOT_SIGN_KEYNAME` (check the complete list in `tdx-signed-fit-image.inc`)) are set through override `k3` to ensure the values coming from layer `meta-toradex-security` override those from layer `meta-ti-bsp`; this should be a temporary measure until fixes to the latter are upstreamed. Due to this, the recommended way to set those variables at the moment is via override `forcevariable`.
+
```

Comment:
Maybe this can be done together with https://github.com/toradex/meta-toradex-security/pull/73/commits/d39f38a44fa462af7b914d832e01c6f4869f0018, so it can be easily reverted later?

Reply:
Given the status of our upstream patch, we'll probably need to review our position here. We could make this a more definitive solution with a known issue or we could consider that idea of not setting the signing-related variables in our layer.

Reply:
To keep consistency between all SoC platforms, I would prefer to have the defaults configured in our layer (using an override) and then documenting how the user can override it via the `forcevariable` override.

Reply:
That sounds sensible to me. The latest version is following that approach already. The metadata changes and corresponding documentation changes were placed into the two sequential commits:

- [97d7a00](https://github.com/toradex/meta-toradex-security/pull/73/commits/97d7a004d5910308d7bf9bc2a22ecc6851864b33) README-secure-boot: list known issues for TI (verdin-am62)
- [e7009cf](https://github.com/toradex/meta-toradex-security/pull/73/commits/e7009cf11a7e58a60e91936d6929a0d1b2dd3f4d) tdx-signed-fit-image: override signing parameters from TI

just in case we ever want to revert this, even though this is unlikely to happen.